letsencrypt-auto-source/letsencrypt-auto.template

```diff
@@ -688,6 +688,8 @@ UNLIKELY_EOF
   fi
 
   if [ "$INSTALL_ONLY" = 1 ]; then
+    # If you're modifying any of the code after this point in this `if` block, you may
+    # need to update the "$DEPRECATED_OS" = 1 case at the beginning of phase 2 as well.
```

Comment:
I should have been more clear in my initial comment, but I was actually imagining we move this comment up a level and put it before the `if [ "$INSTALL_ONLY" = 1 ]; then` line. If we change anything after this point such as how we invoke Certbot, we probably need to update the deprecation case as well.

Reply:
Indeed, it can also impact in non install-only mode.